src/index.ts

```diff
@@ -150,6 +150,9 @@ export {
 export { default as LayoutTabs } from './components/LayoutTabs'
 export { default as Select } from './components/Select'
 export { default as Collapse, type CollapseProps } from './components/Collapse'
+
+export { default as MetaSection } from './components/MetaSection'
```

Comment:
might be good to export `MetaSectionProps` too

Reply:
done